python/src/uagents/context.py

```diff
@@ -349,7 +349,8 @@ async def experimental_broadcast(
         agents = self.get_agents_by_protocol(destination_protocol, limit=limit)
         if not agents:
             self.logger.error(f"No active agents found for: {destination_protocol}")
-            return
+            return []
```

Comment:
My pycharm type checker was not a fan of this @Archento please check that I am not breaking something